packages/plugin-commands-setup/src/setup.ts

```diff
@@ -49,21 +50,18 @@ export async function handler (
     pnpmHomeDir: string
   }
 ) {
-  const currentShell = process.env.SHELL ? path.basename(process.env.SHELL) : null
+  const currentShell = typeof process.env.SHELL === 'string' ? path.basename(process.env.SHELL) : null
```

Comment:
For better typing

```suggestion
  const currentShell = typeof process.env.SHELL !== 'undefined' ? path.basename(process.env.SHELL) : null
```

Reply:
I don't think provided suggestion is a better solution. Current statement clearly indicates **intent** - use ```SHELL``` value only if it is of type ```string```. This makes it very clear for future development _and_ provides runtime type safety in case somehow ```SHELL``` value is corrupted and changed to array or number.

Are there any benefits for using ```typeof X !== 'undefined'``` in this case, that I'm not aware of (like runtime performance or similar)?

Reply:
Indeed. It was not an objection.